environment.rb

```diff
@@ -0,0 +1,139 @@
+module Environment
+    class Depot
+        attr_accessor :packs
+        def initialize 
+            @packs = {
+                :simple_transportation_pack => {
```

Comment:
It is recommended to use this hash notation instead: `simple_transportation_pack: {`.